l2tdevtools/review_helpers/github.py

```diff
@@ -92,6 +92,34 @@ def CreatePullRequestReview(
 
     return True
 
+  def AssignPullRequest(
+      self, pull_request_number, access_token, assignees):
+    """Assigns a GitHub pull request.
```

Comment:
=> "Adds assignees to a GitHub pull request." or "Sets the assignees to a GitHub pull request."

The role of an assignee is to ....

Reply:
Done